src/sharing/components/ShareByEmail.jsx

```diff
@@ -110,7 +112,7 @@ const ShareTypeSelect = ({ options, onChange }) => (
       classNamePrefix="needsclick react-select"
       components={{ DropdownIndicator, Option }}
       styles={customStyles}
-      defaultValue={options[1]}
+      defaultValue={options[0]}
```

Comment:
Should we use the 'value' of the option? Instead of the index ?

Reply:
Mmh no I don't think so, `react-select` can handle an object as a value!